README.md

```diff
@@ -64,7 +64,15 @@ module.exports = {
 
 ### Loading Scoped Styles
 
-You need to specify scoped flag and loaders for style files such as `style-loader` and `css-loader`. Note that they must be `enforce: post` to inject scope id into styles before they are processed by them.
+For an explanation of scoped styles, see the [vue-loader docs](https://vue-loader.vuejs.org/en/features/scoped-css.html). 
+
+Html and style files need to be imported using the loader syntax: `require("./file.html?style=./file.css")`. You also need modify your webpack config as follows:
```

Comment:
Can we use ES Modules syntax instead of CommonJS since it would more standard way in nowadays?